test_env.py

```diff
@@ -33,9 +34,12 @@ def setup_class(cls):
         result = kube_env.setup_bookinfo_deployment("MK", False)
         assert result == util.EXIT_SUCCESS
 
-
     def test_count(self):
-        result = query_tests.test_count()
+        result = query_tests.test_count("MK")
```

Comment:
Why these two tests specifically?  Shouldn't we either do all the old cql queries that worked before, or just the height and get service name?

Reply:
Well this was the attempt to run CI on minikube, but it ultimately failed because of the spontaneous errors. Now minikube tests are disabled. However, I would like to keep these around because we should still be able to support them with the new compiler. Once we have actual working queries, we can add more query-specific tests.

Reply:
Makes sense